### PR TITLE
check err

### PR DESCRIPTION
**Pull request by yu8833**
Signed-off-by: yupengzte <yu.peng36@zte.com.cn>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/cloudprovider/providers/gce/gce.go [hunk 1]
```diff
@@ -358,3 +361,4 @@ func newOauthClient(tokenSource oauth2.TokenSource) (*http.Client, error) {
 
 	return oauth2.NewClient(oauth2.NoContext, tokenSource), nil
 }
+}
```
Comment (gmarek) [inline pkg/cloudprovider/providers/gce/gce.go]: This looks wrong.

Comment (YuPengZTE) [inline pkg/cloudprovider/providers/gce/gce.go]: @gmarek , are you mean it is same with [client, err := newOauthClient(tokenSource)](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/gce/gce.go#L179)?

Comment (gmarek) [inline pkg/cloudprovider/providers/gce/gce.go]: No - I meanadding additional closing bracket.

Comment (zhangxiaoyu-zidif) [inline pkg/cloudprovider/providers/gce/gce.go]: Hi, @gmarek , is it right to delete line 189?

Comment (gmarek) [inline pkg/cloudprovider/providers/gce/gce.go]: No, we need separate alpha client. I guess - I'm not the one who wrote this code. @bowei

**Review by gmarek — CHANGES_REQUESTED** [review]

**Comment by YuPengZTE** [comment]
@gmarek , I know the useage, Thanks. Close it.

**Pull request closed without merging**